### PR TITLE
update renovate config

### PR DESCRIPTION
**Pull request by juliangieseke**
lets try again…

## Trade-offs

`dependencies` that are behind in `major` are pinned to that `major` by only enabling `patch` and `minor` updates
same for all `devDependencies`

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
+        "clipboard",
+        "d3",
+        "flux",
+        "graphql-tools",
+        "inversify",
+        "query-string",
+        "react-ace",
+        "react-bootstrap-typeahead",
+        "react-router",
+        "redux"
```
Comment (pierrebeitz) [inline renovate.json]: i never used renovate before. do i get correctly that it'll only create PRs for the packages declared in `packageNames`? if so, how is this list compiled?

Comment (juliangieseke) [reply renovate.json]: no, this config applies special settings to these packages - I added them manually. Its a list of things that are behind in major version (we're using redux v3, but most recent is v5)

Idea is to manually update these packages and remove them from this list.

Comment (juliangieseke) [reply renovate.json]: everything thats not in this list will be updated to `latest` by renovate.

Comment (pierrebeitz) [reply renovate.json]: now i also understand the things in the trade-offs-section. thanks!

**Pull request merged**